dataset/dictionary/z-archives.json5

```diff
@@ -209,54 +238,62 @@
   {
     en: "Sundered Feather",
     ja: "切落の羽",
+    zhCN: "切落之羽",
     notes: "聖遺物セット「絶縁の旗印」(Emblem of Severed Fate) の1つ",
     tags: [ "artifact-piece" ],
   },
   {
     en: "Storm Cage",
     ja: "雷雲の印籠",
+    zhCN: "雷云之笼",
     pronunciationJa: "らいうんのいんろう",
     notes: "聖遺物セット「絶縁の旗印」(Emblem of Severed Fate) の1つ",
     tags: [ "artifact-piece" ],
   },
   {
     en: "Scarlet Vessel",
     ja: "緋花の壺",
+    zhCN: "绯花之壶",
     notes: "聖遺物セット「絶縁の旗印」(Emblem of Severed Fate) の1つ",
     tags: [ "artifact-piece" ],
   },
 
   {
     en: "Exile's Flower",
     ja: "亡命者の花",
+    zhCN: "",
```

Comment:
You forget adding translation here.
(But strange, it should be pointed out by CI. I'll check it later.)

Reply:
Oops, I fixed it.